Criollo/Source/HTTP/CRHTTPResponse.m

```diff
@@ -62,7 +62,7 @@ - (void)buildHeaders {
 
 - (void)writeData:(NSData *)data finish:(BOOL)flag {
     if ( self.finished ) {
-        @throw [NSException exceptionWithName:NSInternalInconsistencyException reason:@"Response is already finished" userInfo:nil];
```

Comment:
This is on purpose. On should never get into a state where data is being sent after the response has finished.  It's mean to point out application logic errors.

We should leave it as it.